R/score.R

```diff
@@ -76,7 +76,7 @@ score.default <- function(data, ...) {
 
 #' @rdname score
 #' @export
-score.forecast_binary <- function(data, metrics = metrics_binary, ...) {
+score.forecast_binary <- function(data, metrics = rules_binary(), ...) {
```

Comment:
do these need to be scoringutils::rules_binary etc to work when scoringutils isn't loaded?

Reply:
The following works fine even with `scoringutils` not loaded:

`scoringutils::score(data.frame(predicted = 1:10, observed = 1:10, id = 1:10))`

So I think it should be fine

Reply:
hmm interesting. well seems like magic to me